main.js

```diff
@@ -65,7 +76,55 @@ function displayMenu() {
     document.getElementById("timer").style.visibility = "visible";
 }
 
-// creating the countdown timer depending on the duration (number of minutes)
+// calculate the numbers of bombs around each cell
+function getDistanceBomb(size) {
```

Comment:
Let's be consistent and keep the same name for all variables/parameters with the same goal.
Here you can use `tableSize` instead of `size`.